src/types/carrier.ts

```diff
@@ -21,7 +21,7 @@ export enum CarrierName {
   Dpd = 'dpd',
   Instabox = 'instabox',
   Dhl = 'dhl',
-  BolCom = 'bol.com',
+  Bol = 'bol.com',
```

Comment:
Oeps, dit is een breaking change, ik zou BolCom terug zetten onder deze, en als deprecated markeren met `will be removed in the next version` oid